docs/front-matter.md

```diff
@@ -0,0 +1,195 @@
+
+`layout: page-breadcrumbs.html`
+
+### Template
+
+There are 3 standard template options for content:
+
+| Template                   | Use                                                                                                                                                                                                                                                                                                                                                                                      |
+|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `template: level2-index `  | Used for hub pages like [va.gov/education/index.html](https://www.va.gov/education/). This template can display content and link lists, right-side promo and social contact links, but no left-side accordian navigation.                                                                                                                                                                |
+| `template: detail-page `   | Used for detail pages like [va.gov/education/about-gi-bill-benefits/](https://www.va.gov/education/about-gi-bill-benefits/). Displays the same elements as `topic-landing`, in addition to previous/next page navigation at the bottom of the page to navigate through sequential content. Can also be used for landing pages that would benefit from the previous/next page navigation. |
+| `template: topic-landing ` | Used for deeper landing pages like [va.gov/disability/eligibility/special-claims/](https://www.va.gov/disability/eligibility/special-claims/). This template can display content, side navigation, main links, and related links.                                                                                                                                                        |
+
+### Title vs. Display Title vs Heading
+
+**Title** is the full title displayed at the top of the page. Example:
+
+`title: VA Burial Benefits and Memorial Items`
+
+It is also what Google uses for SEO meta content. 
+
+**Display title** is an optional shorter version of the title that appears in the side navigation and breadcrumbs. If left blank, these locations will display the content in the `title` field. Example:
+
+`display_title: Burials and Memorials`
+
+**Heading** is currently a conditional front matter attribute. When a value is defined, it will display as the title at the top of the page. If left empty, the title will be used to display at the top of the page.
```

Comment:
Can we also add a word or two here saying that this is the large font that's actually visible on the page? This and title both say "at the top of the page" so it's hard to draw the distinction.

Reply:
changed description to say "is currently a conditional front matter attribute. When a value is defined, it will display as the <h1> heading visable at the top of the page. If left empty, the title will be used to display as the <h1> heading."